src/server/dash-server.ts

```diff
@@ -204,9 +194,61 @@ export class DashServer {
 
         outgoingPrs.push(object);
       }
+
+      // Incoming reviews
+      for (const pr of result.data.incomingReviews.nodes || []) {
+        if (!pr || pr.__typename !== 'PullRequest') {
+          continue;
+        }
+
+        const object: IncomingPullRequest = {
+          ...prFieldsToResult(pr),
+          myReview: null,
```

Comment:
What is this used for?

Reply:
This fields contains your latest review.